### PR TITLE
RCF-203,205 & 211

### PR DESCRIPTION
**Pull request by aviral-mishra-hunter**
[RCF-203] [RCF-205] [RCF-211] [RCF-223] [RCF-224] [RCF-225]

[RCF-203]: https://mosip.atlassian.net/browse/RCF-203?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[RCF-205]: https://mosip.atlassian.net/browse/RCF-205?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[RCF-211]: https://mosip.atlassian.net/browse/RCF-211?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[RCF-223]: https://mosip.atlassian.net/browse/RCF-223?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[RCF-224]: https://mosip.atlassian.net/browse/RCF-224?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[RCF-225]: https://mosip.atlassian.net/browse/RCF-225?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: lib/ui/login_page.dart [hunk 1]
```diff
@@ -1,27 +1,21 @@
+// ignore_for_file: use_build_context_synchronously
```
Comment (jainhitesh9998) [inline lib/ui/login_page.dart:1]: any reason for ignoring lint rules here

**Review by jainhitesh9998 — CHANGES_REQUESTED** [review]

File: android/clientmanager/src/main/java/io/mosip/registration/clientmanager/entity/Registration.java [hunk 2]
```diff
@@ -101,7 +101,7 @@ public class Registration {
 
     @Override
     public String toString() {
-        return packetId + "\n" + (serverStatus == null ? clientStatus : serverStatus);
+        return packetId + " " + (serverStatus == null ? clientStatus : serverStatus);
```
[the remark this answers is not in the document]
Comment (developer-aditya) [reply android/clientmanager/src/main/java/io/mosip/registration/clientmanager/entity/Registration.java:104]: I was trying to log all packet Id with status in single line. we can change it back to \n

File: lib/platform_android/packet_service_impl.dart [hunk 3]
```diff
@@ -31,6 +31,20 @@ class PacketServiceImpl implements PacketService {
...
   }
+
+  @override
+  Future<List<String?>> getAllRegistrationPacket() async {
+    List<String?> packetStatus = [];
+
+    try {
+      packetStatus = await PacketAuthApi().getAllRegistrationPacket();
+    } on PlatformException {
+      debugPrint('PacketAuthenticationApi call failed!');
```
[the remark this answers is not in the document]
Comment (developer-aditya) [reply lib/platform_android/packet_service_impl.dart:42]: Log doesn't show up when we run the app in release mode.

File: android/app/src/main/java/io/mosip/registration_client/MainActivity.java [hunk 4]
```diff
@@ -135,9 +147,94 @@ public class MainActivity extends FlutterActivity {
...
+        );
+    }
+
+    private void fetchRegistrationPackets(Context context) {
+        if(NetworkUtils.isNetworkConnected(context)){
+            Log.d(getClass().getSimpleName(), "Fetching Packets in main activity");
+            List<Registration> registrationList = packetService.getAllRegistrations(1,5);
+            Log.e(getClass().getSimpleName(), "Registration : "+ registrationList);
+
+            registrationList = packetService.getRegistrationsByStatus("CREATED");
```
[the remark this answers is not in the document]
Comment (developer-aditya) [reply android/app/src/main/java/io/mosip/registration_client/MainActivity.java:218]: Okay, will work on this

**Review by jainhitesh9998 — APPROVED** [review]

**Pull request merged**